docs/en/reference/index.rst

```diff
@@ -233,45 +233,9 @@ AnnotationDriver
 
 .. note::
 
-    This driver requires the ``doctrine/annotations`` project. You can install it with composer.
+    This driver requires the ``doctrine/annotations`` project and is
+    deprecated because of that.
```

Comment:
I'd add a note about attributes superseding annotations

Reply:
Also why removing the docs below? The feature is deprecated but documentation is still valuable :)

Reply:
I think it feels weird to deprecate a feature yet help people build on it. I thought I would make it harder for people to shoot themselves in the foot.